go/vt/schema/parser.go

```diff
@@ -71,6 +72,19 @@ func ReplaceTableNameInCreateTableStatement(createStatement string, replacementN
 	return fmt.Sprintf("%s`%s`%s", submatch[1], replacementName, submatch[3]), nil
 }
 
+// ReplaceViewNameInCreateViewStatement returns a modified CREATE VIEW statement, such that the view name is replaced with given name.
+// This intentionally string-replacement based, and not sqlparser.String() based, because the return statement has to be formatted _precisely_,
+// up to MySQL version nuances, like the original statement. That's in favor of tengo view comparison.
+// We expect a well formatted, no-qualifier statement in the form:
+// CREATE TABLE `some_table` ...
```

Comment:
Isn't this supposed to say `CREATE VIEW ...`?

Reply:
Good function description :)

Reply:
The function uses a different regular expression than `ReplaceTableNameInCreateTableStatement`.

I have a plan to ultimately remove both functions in the future, if all goes well.